### ISSUE TITLE
React Native image borderRadius not working.

### ISSUE THREAD

**Issue by rkobhi512** [facebook/react-native#28496]
Image borderRadius/Native base Thumbnail circular shape/Gifted chat showUserAvatar={true}/any type of circular images are not working in Android 5.1 Lollipop. 

**Details**
I am trying to display an image from a URL. In the emulator image displays perfectly. When I run this on the device it displays a black block and after some moment the app crashes without any error. (I have reveiw the adb logcat)

If I remove borderRadius property from Image style it works perfectly. But I need the circular shape.

I tried to native-base Thumbnail instead of Image but the problem persists. Then I tried to make a circular shape using View with overflow:hidden property but image edge not smooth. How can I solve this?

**Screenshots**
_Black Block image in the device_
![black block](https://user-images.githubusercontent.com/31918004/78274553-3965aa00-7532-11ea-8057-4a35d23f41d2.png)

_Example of code_
![ProCode](https://user-images.githubusercontent.com/31918004/78275205-1b4c7980-7533-11ea-95c9-007e36e029c0.JPG)

Please pardon any mistake and help me.

**Comment by rkobhi512** [comment]
react-native info command not work.
_

> C:\Users\Name\MyProject>react-native info
> info Fetching system and libraries information...
> error Unable to print environment info.
> Error: Command failed: wmic os get Caption
> 'wmic' is not recognized as an internal or external command,
> operable program or batch file.

_

react-native -v command:
**react-native-cli: 2.0.1
react-native: 0.61.5**

**Comment by GuilhermeTrivilin** [comment]
Try to wrap the image with a view with the dimensions you want and, in the image, put these styles to resize, like this:

```
return (
        <View
            style={styles.container}
        >
            <Image
                style={styles.image}
                source={{ uri: <LINK OF IMAGE> }}
            />
        </View>
    )
}

const styles = StyleSheet.create({
    container: {
        height: 150,
        width: Dimensions.get('window').width / 3,
        marginHorizontal: 5,
        justifyContent: 'center',
        alignItems: 'center',
    },
    image: {
        flex: 1,
        width: null,
        height: null,
        resizeMode: 'contain'
    },
})
```

If it doesn't work, try replacing the image style with:

`image: {  
        width: '100%',
        resizeMode: 'contain',
        borderRadius: 60
 }`

**Comment by rkobhi512** [comment]
I have try this. Image not display both emulator and device.

**Comment by rkobhi512** [comment]
The problem is solved.
The main problem was Google Cloud Storage where I stored the pictures.

Having trouble using various features of Google Cloud Platform from my country.

The black blocks we see here are basically loading from cache memory. So these are showing black block in circular shape. Because, images are saved in cache memory with square shape.

I have tried in VPN and everything is ok.
Thank you.

**Comment by rkobhi512** [comment]
The problem is solved.
The main problem was Google Cloud Storage where I stored the pictures.

Having trouble using various features of Google Cloud Platform from my country.

The black blocks we see here are basically loading from cache memory. So these are showing black block in circular shape. Because, images are saved in cache memory with square shape.

I have tried in VPN and everything is ok.
Thank you.
